### PR TITLE
Added button in UI to link Zulip

### LINKED ISSUE #1038: Add button in UI to link Zulip
## Description
As a user, I need to add a button to link our Zulip communication platform, so that I can have smooth communication through project.

### Update [Required]
- [ ] Screen shot in PR to show Button added
- [ ] Link to organisation group, Zulip

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
2 hours

### PR DESCRIPTION
**Pull request by yash2189**
# Description
To add button in UI to link Zulip
Fixes #1038

# Type of Change:
- User Interface

# How Has This Been Tested?
Successfully added button in UI to link Zulip. When a registered user is logged in the button is shown on the home page.

Screenshots:
Button_ZULIP

# Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] Any dependent changes have been merged

### CODE REVIEW

File: vms/home/templates/home/home.html [hunk 1]
```diff
@@ -20,6 +20,10 @@
     <div class="jumbotron jumbotron-custom">
         <h1 class="index">Volunteer Management System</h1>
         <p>{% trans "Systers, an Anita Borg Institute Community, has many volunteers that devote their time, skills and commitment to our vision. Volunteer Management System serves as a place where volunteers can view volunteer events, sign up for volunteer jobs and log volunteer hours. Volunteer Management System also enables volunteer managers to manage their volunteers and run volunteer related reports." %}</p>
+	 {% if user.is_authenticated %}
+        <a href="http://anitab-org.zulipchat.com">
+          <button type="button" class="btn btn-primary btn-lg">Connect With us 			on Zulip</button></a>
```
Comment (keshakaneria) [inline vms/home/templates/home/home.html:25]: There seems an unnecessary spacing after 'us'. You need to remove it.

Comment (keshakaneria) [inline vms/home/templates/home/home.html:25]: Also check if the button's position and size is responsive or not.

**Review by keshakaneria — CHANGES_REQUESTED** [review]

**Comment by yash2189** [comment]
@keshakaneria  I have checked it please review it again.

**Comment by keshakaneria** [comment]
Also can you let me know why was the images added in docs/screenshots file?

**Comment by yash2189** [comment]
I thought the screenshot would be required hence I did. Does it have to be removed or placed in different folder?

**Comment by keshakaneria** [comment]
> I thought the screenshot would be required hence I did. Does it have to be removed or placed in different folder?
> However I have placed them in a separate folder now called UI_screenshots.
> If this is fine?

Well we don't need any screenshots to be uploaded in the repo how the UI looks. So it is better if we dont add any of the screenshots in commits. There should be only the code changed in `home.html` and nothing else. So remove all those screenshots and push it for the final review.
Points to change:
- Remove the screenshots
- Squash the final commits
Hence it will be ready to merge. Otherwise great job @yash2189 ! 👍 :)

**Comment by yash2189** [comment]
Surely @keshakaneria I'll do that by tomorrow if that works?

**Comment by yash2189** [comment]
@keshakaneria would you please tell me how do I squash those final commits , I haven't used the squash command before.

**Comment by keshakaneria** [comment]
> @keshakaneria would you please tell me how do I squash those final commits , I haven't used the squash command before.

Refer this for squashing https://www.internalpointers.com/post/squash-commits-into-one-git. You'll get to know how it is done.

**Comment by yash2189** [comment]
@keshakaneria  I have squashed the commits.

**Comment by keshakaneria** [comment]
> @keshakaneria I have squashed the commits.
> Would you please check whether I have done it correctly?

The commits count still shows me 4. After squashing it will change to a single commit. Try once more if you are able to do.

**Comment by yash2189** [comment]
Can I please close this PR and make a new PR instead I'd be able to do that quick

**Comment by keshakaneria** [comment]
I guess this is not good for you to open 3 PRs for an individual issue. Take care next time if any issue assigned to you.

**Comment by yash2189** [comment]
Apologies,I will improve on this surely.

**Comment by yash2189** [comment]
@keshakaneria  Can you please review this PR #1091 ?

**Pull request closed without merging**